src/controllers/auth.js

```diff
@@ -1,11 +1,94 @@
+const jwt = require('jsonwebtoken');
+
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+
+const User = require('../models/user');
+
 module.exports = {
-  signInUser: (req, res) => {
-    res.send('You successfully signed in');
+  signInUser: async (req, res) => {
+    res.json('');
```

Comment:
Why is this function commented out?

Reply:
Because I needed just signup functionality, but since it got assigned to me I will uncomment and work on it.